docs/concepts/whatisvpp/ndrrates.rst

```diff
@@ -0,0 +1,53 @@
+.. _ndrrates:
+
+.. toctree::
+    
+NDR Rates 
+***********************************
+
+NDR rates for 2p10GE, 1 core, L2 NIC-to_NIC
```

Comment:
NDR rates for 2 ports, 1 core, 1 thread, L2 NIC-to-NIC
remove reference to 10GE